examples/hn.svelte.dev/.netlify/netlify-plugin-pnpm/index.js

```diff
@@ -1,8 +1,9 @@
-export default {
+module.exports = {
```

Comment:
Why is this being changed to CJS?

Reply:
The Netlify build system doesn't accept ESM for plugins because it `require()`s them from CJS: https://github.com/netlify/build/blob/main/packages/build/src/plugins/child/logic.js#L14